pservice/pdo/pservice/scripts/PServiceCLI.py

```diff
@@ -503,7 +503,7 @@ def Main() :
     # set up the ledger configuration
     if config.get('Ledger') is None :
         config['Ledger'] = {
-            'LedgerURL' : 'http://localhost:8008',
+            'LedgerURL' : 'http://localhost:6600',
```

Comment:
does this even work without your name based changes for ccf?

Reply:
Good point, and it's not easy to support default values here.

These default values are probably not used with CCF, and the url must be passed in the configuration.

I'll just raise an error if no url is passed.